Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using System;
+using Newtonsoft.Json;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.EntityFrameworkCore; 
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Configuration.AddJsonFile("appsettings.json");
+
+var connectionString = builder.Configuration.GetConnectionString("MyAppDatabase");

```

Comment:
Rename "MyAppDatabase" to "HolyShiftDbConfig.

Also, it should not be the connection string it should be separated fields and let options.UseMySql join them into the connection string.